### DISCUSSION TITLE
How to find the right location for streamlink to set with streamlink gui

### DISCUSSION THREAD

**Discussion by tebrow18** [streamlink/streamlink-twitch-gui#822] [General]
I am trying to find the right location for pip when I installed streamlink I need to located the right path to place streamlink in the streamlink gui settings **Streaming** an I place streamlink I found the path of were the pip installed streamlink /root/.local/lib/python3.8/site-packages/streamlink (but I don't know which py file to choice)

![Screenshot from 2021-08-20 14-01-40](https://user-images.githubusercontent.com/89272476/130275061-5fbe1e31-d6aa-4473-8210-13b0bd278412.png)

**Comment by bastimeyer** [comment]
When installing Streamlink via `pip --user`, python entry scripts get written to `~/.local/bin`. If you add this path to your `PATH` env var, which you should do, then you don't need to add `~/.local/bin/streamlink` to the Twitch GUI's streaming settings menu and it'll pick it up automatically. Adding `~/.local/bin` to `PATH` also allows you to run `streamlink` from your command line shell.

However, you seem to have run `pip` with your system's root account instead of your user account. You should fix that.
